wercker.yml

```diff
@@ -59,3 +63,46 @@ push:
         tag: $VERSION
         entrypoint: /oci-cloud-controller-manager
         user: 65534 # nobody
+
+release:
+  box:
+    id: oraclelinux:7-slim
+  steps:
+    - script:
+        name: set ENV vars
+        code: |
+          export VERSION=$(cat VERSION.txt)
+          echo "${VERSION}"
+          
+    - script:
+        name: prepare
+        code: |
+          mv ./oci-cloud-controller-manager /oci-cloud-controller-manager
+          chmod +x /oci-cloud-controller-manager
+
+    - internal/docker-push:
+        repository: wcr.io/oracle/oci-cloud-controller-manager
+        tag: $GITHUB_RELEASE_VERSION
+        entrypoint: /oci-cloud-controller-manager
+        user: 65534 # nobody
+
+    - github-create-release:
+      token: $GITHUB_TOKEN
+      tag: $GITHUB_RELEASE_VERSION
+      title: Application $GITHUB_RELEASE_VERSION
+      draft: false
+
+    - github-upload-asset:
+      token: $GITHUB_TOKEN
+      file: $WERCKER_OUTPUT_DIR/oci-cloud-controller-manager.yaml
+      filename: oci-cloud-controller-manager.yaml
+      content-type: text/yaml
+
+    - github-upload-asset:
+      token: $GITHUB_TOKEN
+      file: $WERCKER_OUTPUT_DIR/oci-cloud-controller-manager-rbac.yaml
```

Comment:
Same as above. 

$WERCKER_OUTPUT_DIR doesn't contain the previous pipelines input. That should be on the filesystem in current directory. 

file: oci-cloud-controller-manager-rbac.yaml